ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -255,6 +252,39 @@ private void validateConfig(final JsonRpcConfiguration config) {
     return resultFuture;
   }
 
+  private HttpServerOptions getHttpServerOptions() {
+    final HttpServerOptions httpServerOptions =
+        new HttpServerOptions()
+            .setHost(config.getHost())
+            .setPort(config.getPort())
+            .setHandle100ContinueAutomatically(true);
+    config
+        .getTlsConfiguration()
+        .ifPresent(
+            tlsConfiguration -> {
+              httpServerOptions
+                  .setSsl(true)
+                  .setPfxKeyCertOptions(
+                      new PfxOptions()
+                          .setPath(tlsConfiguration.getKeyStore().getStorePath())
+                          .setPassword(tlsConfiguration.getKeyStore().getStorePassword()));
+
+              tlsConfiguration
+                  .getTrustStore()
+                  .ifPresent(
+                      trustStore -> {
+                        httpServerOptions
+                            .setClientAuth(ClientAuth.REQUIRED)
+                            .setPfxTrustOptions(
+                                new PfxOptions()
+                                    .setPath(trustStore.getStorePath())
+                                    .setPassword(trustStore.getStorePassword()));
+                      });
+            });
```

Comment:
although its technically less than 3 lines, probably worth breaking lambda out to a separate function